mjpeg2sd.cpp

```diff
@@ -585,6 +585,7 @@ void stopPlaying() {
   if (isPlaying) {
     // force stop any currently running playback
     stopPlayback = true;
+    controlFrameTimer(true);

```

Comment:
When `control?stopStream=1` is sent while an SD file playback is streaming, it almost always results in a ~10s delay before the playback could non-forcibly finish (and the HTTP request subsequently terminate). Note that I did these tests by replacing  the `while (isPlaying ...)` mechanism with waiting on a binary mutex which was released after clean playback finish.

Resetting the frame timer here seems to fix this but I haven't figured exactly what's going on. Would be glad to hear your opinion.